virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,43 @@
+"""
+libvirt related misc functions
+"""
+import logging
+
+from avocado.core import exceptions
+
+from virttest import libvirt_version
+
+
+def is_libvirt_feature_supported(params, ignore_error=False):
+    """
+    Check whether the function is supported in this libvirt version by comparing
+    the installed libvirt version
+
+    :param params: Dictionary with the test parameters
+    :param ignore_error: Whether to raise an exception
+    :raise: When ignore_error is set False, raise TestCancel if the feature is
+        not supported
+    :return: True if the feature is supported;
+        False if the feature is not supported when ignore_error is set to True
+
+    Example:
+    ::
+    params={'func_supported_since_libvirt_ver':'(6,8,0)'}
+
+    NOTE: The value of 'func_supported_since_libvirt_ver' is a string of
+        libvirt's (major, minor, update) version.
+    """
+    func_supported_since_libvirt_ver = eval(
+        params.get("func_supported_since_libvirt_ver", '()'))
+    unspported_err_msg = params.get("unspported_err_msg",
```

Comment:
The error msg is too generalized, better output specific feature

Reply:
You can configure it as you want by passing unspported_err_msg.